libbeat/feature/feature.go

```diff
@@ -0,0 +1,120 @@
+package feature
+
+import (
+	"fmt"
+	"reflect"
+)
+
+//go:generate stringer -type=Stability
```

Comment:
How about the 'generate stringer' command next to the type declaration or in a separate `0gen.go` file. The generated file already contains the type name. Having the generate command next to the type declaration makes it easier to find when navigating the code. Like:

```
// Comment...
// go:generate stringer -type=Stability
type Stability int
```

Reply:
This was added automatically when you call `generate stringer -type=Stability` in the feature directory :)